Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM openjdk:8-jre-alpine
+MAINTAINER "Manojv" "manojv@ilimi.in"
+RUN apk update \
+    && apk add  unzip \
+    && apk add curl \
+    && adduser -u 1001 -h /home/sunbird/ -D sunbird \
+    && mkdir -p /home/sunbird/learner
+#ENV sunbird_learnerstate_actor_host 52.172.24.203
+#ENV sunbird_learnerstate_actor_port 8088 
+COPY ./service/target/telemetry-service-1.0-SNAPSHOT-dist.zip /home/sunbird/learner/
+RUN unzip /home/sunbird/learner/telemetry-service-1.0-SNAPSHOT-dist.zip -d /home/sunbird/learner/
+RUN chown -R sunbird:sunbird /home/sunbird
+USER sunbird
+WORKDIR /home/sunbird/learner/
```

Comment:
Check with devops if the logs folder needs to be "learner" or "telemetry"?

Reply:
Done